src/components/ProfileCard/index.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Text from "../Text";
+import { LogoTwitter, LogoLinkedin, LogoGithub } from "@carbon/icons-react";
+
+const ProfileCard = ({ photoUrl, name, role, twitter, linkedin, github }) => {
+  return (
+    <div className="flex rounded-[4px] bg-light-on-background-200 max-w-max">
+      <img src={photoUrl} alt={name} className="rounded-l-[4px]" />
+      <div className="p-10 py-20">
```

Comment:
![2022-12-19_14-50-11](https://user-images.githubusercontent.com/27215/208430161-2f5c654c-6189-44dd-92c0-6f0d45223899.png)

Can we make so that for mobile devices the text part goes under the photo? Achievable with responsive grid too.